### ISSUE TITLE
SS4. Minor. New site title not displayed at top left after "Save".

### ISSUE THREAD

**Issue by SpiritLevel** [silverstripe/silverstripe-siteconfig#102]
If you edit Site title in Settings and click "Save", the Site title appearing in the top left of the CMS is not updated; it does get updated upon page reload, though.

**Comment by maxime-rainville** [comment]
I guess the problem here is that saving the SiteConfig is done via an AJAX call and the site title is loaded via rendered templates.

My thinking here is that this is probably not worth fixing:
* This would involved adding some clunky hook to this single form ;
* This not something most editors will do very often or have access to do ;
* The change gets reflected correctly as soon as you navigate to a different controller.

I'm happy to hear a contrarian view.

**Comment by kinglozzer** [comment]
I’m sure we used to use an HTTP header (something along the lines of `X-Reload: 1`) to signal that the browser should perform a hard refresh after saving... then again perhaps I dreamt it 😅

**Comment by robbieaverill** [comment]
Yeah, we could use a PJAX fragment reload to achieve this if we wanted to. I agree with the triaging though that it's pretty low impact.
